wagtailio/project_styleguide/templates/patterns/includes/search.html

```diff
@@ -0,0 +1,23 @@
+<div class="modal modal--search micromodal-slide" id="site-search-modal" aria-hidden="true">
+    <div class="modal__overlay" tabindex="-1" data-micromodal-close>
+        <div class="modal__container" role="dialog" aria-modal="true">
+            <main class="modal__content">
+                <button class="modal__close-button" aria-label="Close the Search" data-micromodal-close>
+                    <svg class="modal__close-icon" aria-hidden="true">
+                        <use xlink:href="#cross"></use>
+                    </svg>
+                </button>
+                <form action="#" method="get" role="search" class="search">
+                    <div class="search__wrap">
+                        <label class="u-sr-only" for="search-page">Enter a search term</label>
+                        <input class="search__input" id="search-page" type="text" name="query"{% if search_query %} value="{{ search_query }}"{% endif %}>
```

Comment:
![image](https://user-images.githubusercontent.com/22497837/183664481-3c7961a1-0cf7-4028-9719-d3002e99ee52.png)
Looks like parts of a `g` character will get cut off, I think `height: 100%` would sort this

Reply:
Thanks!